x-pack/plugins/apm/server/lib/backends/get_throughput_charts_for_backend.ts

```diff
@@ -81,7 +78,10 @@ export async function getThroughputChartsForBackend({
     response.aggregations?.timeseries.buckets.map((bucket) => {
       return {
         x: bucket.key + offsetInMs,
-        y: bucket.throughput.value,
+        y: calculateThroughputWithInterval({
+          bucketSize,
+          value: bucket.doc_count,
+        }),
```

Comment:
@cauemarcondes It looks like there is a bug here. You removed the rate agg on `SPAN_DESTINATION_SERVICE_RESPONSE_TIME_COUNT` and replaced it with a rate calculation on `doc_count`. I don't think this is correct.

If there is a bug, I don't think the worst part is the bug itself, but that we yet again don't have good enough test coverage to spot these things. It still feels way too easy to gloss over these details and sneak a bug in.